src/rocksdb_options.rs

```diff
@@ -949,6 +974,10 @@ impl Options {
             crocksdb_ffi::crocksdb_options_set_allow_concurrent_memtable_write(self.inner, v);
         }
     }
+
+    pub fn set_env(&mut self, env: &Env) {
```

Comment:
We need to share Env among two or more db instance, but Options cannot be shared. We should keep Env's lifetime as the same to databases.